lib/rubocop/cop/naming/block_forwarding.rb

```diff
@@ -90,21 +90,11 @@ def block_argument_name_matched?(block_pass_node, last_argument)
           last_argument.source == block_pass_node.source
         end
 
-        # Prevents the following syntax error:
-        #
-        # # foo.rb
-        # def foo(&)
-        #   block_method do
-        #     bar(&)
-        #   end
-        # end
-        #
-        # $ ruby -vc foo.rb
-        # ruby 3.3.0 (2023-12-25 revision 5124f9ac75) [x86_64-darwin22]
-        # foo.rb: foo.rb:4: anonymous block parameter is also used within block (SyntaxError)
-        #
+        # Ruby 3.3.0 had a bug where accessing an anonymous block argument inside of a block
```

Comment:
Maybe this should also be mentioned in the cop's user docs?

Reply:
Sure, I added a note to both cops